src/routes/docs/products/sites/domains/+page.markdoc

```diff
@@ -0,0 +1,55 @@
+---
+layout: article
+title: Domains
+description: Discover how domains can be managed for an Appwrite Site
+---
+
+Each deployed site can have its own domain, which can be Appwrite-generated or custom. You can use this domain to consume web apps deployed on Appwrite Sites. Appwrite generates TLS certificates to enforce HTTPS on all Appwrite Sites domains. These domains are safe to use and access in production.
+
+[Learn about Sites development >](/docs/products/sites/develop)
+
+# Generated domains
+
+1. In the Appwrite Console's sidebar, click **Sites**.
+2. Under the **Domains** tab, you'll find the domain generated by Appwrite.
+
+The domain usually has this format.
+
+```bash
+https://64d4d22db370ae41a32e.sites.appwrite.global
+```
+
+## Branch and Commit URLs
+
+Additional to the site URL, Appwrite also generates a URL for the branch and commit that your site has been deployed from. The branch URL will remain consistent for all deployments made for code pushed to a specific branch, whereas the commit URL will be updated every time a new deployment is made via the Git integration (i.e. when code is pushed to your repo).
+
+![Branch and Commit URLs](https://warm-tray-285.notion.site/image/attachment%3Aaffbf4bd-be7f-481a-a6ec-4db9e1d49785%3Aimage.png?table=block&id=1cefb53f-e82d-805c-bcbe-f7784ac2ad8f&spaceId=cf760f55-c352-4759-b494-006b8b294dbb&width=1420&userId=&cache=v2)
+
+To find the branch and commit URLs of any deployment:
+
+1. Navigate to your site on Appwrite Console.
+2. Head to the **Deployments** tab and click on any deployment.
+3. In the **Domains** section, click on the **+2** next to the mentioned domain.
+
+# Add a custom domain
```

Comment:
We should have two sets of instructions here. Apex domains will have to use Appwrite's DNS server in the address `appwrite.zone` (this is similar to what used to commonly be ns1.dns.com, n2.dns.com). Subdomains can also use our DNS, but alternatively also CNAME records.

Reply:
I am waiting for our DNS server settings to be added to the QA server. Once that is done, this page will receive a major edit.